### ISSUE TITLE
v1.4.5 compiles - v1.5 not (on Windows 7)

### ISSUE THREAD

**Issue by dreyling** [eudaq/eudaq#151]
I've tried to install EUDAQ v1.5 on the NI crate (Windows 7), but it stopped, see below.. 
As cross check, I have tried using v1.4.5, which was working (0 Errors, 203 Warnings)

CMAKE
```
c:\opt\eudaq\build>cmake -DBUILD_onlinemon=OFF -DBUILD_tlu=ON ..
-- Building for: Visual Studio 12 2013
-- The C compiler identification is MSVC 18.0.21005.1
-- The CXX compiler identification is MSVC 18.0.21005.1
-- Check for working C compiler using: Visual Studio 12 2013
-- Check for working C compiler using: Visual Studio 12 2013 -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler using: Visual Studio 12 2013
-- Check for working CXX compiler using: Visual Studio 12 2013 -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Checking for C++11 compliance of compiler
-- Checking for C++11 compliance of compiler - available
-- Using MSVC-specific preprocessor identifiers and options
-- Main EUDAQ library directory is dirty (uncommitted changes present):
 Process failed because: The system cannot find the file specified

for command: git status --porcelain C:/opt/eudaq/main.
-- Determined EUDAQ version v1.5.0*
-- Looking for include file pthread.h
-- Looking for include file pthread.h - not found
-- Found Threads: TRUE
-- Could NOT find PROTOBUF (missing:  PROTOBUF_LIBRARY PROTOBUF_INCLUDE_DIR)
-- Looking for Root...
-- Found ROOT: C:/root/bin/root (WIN32/version not identified)
-- Looking for CMSPixel dependencies: pxarCore...
-- NOT building converter plugin for CMSPixel.
-- Found Qt5Widgets: C:/Qt/Qt5.3.1/5.3/msvc2013/lib/cmake/Qt5Widgets
-- Found Qt .., will build GUI executables
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Found ZestSC1 package in 'extern' subfolder: C:/opt/eudaq/extern/ZestSC1
-- Found ZestSC1: C:/opt/eudaq/extern/ZestSC1/windows 7/Lib/x86/ZestSC1.lib
-- Found libusb package in 'extern' subfolder: C:/opt/eudaq/extern/libusb-win32-
bin-1.2.6.0
-- Found LIBUSB: C:/opt/eudaq/extern/libusb-win32-bin-1.2.6.0/lib/msvc/libusb.li
b (lib) and C:/opt/eudaq/extern/libusb-win32-bin-1.2.6.0/include (header)
-- Found TLUFirmware: C:/opt/eudaq/extern/tlufirmware
-- Building producer: tlu
-- CMAKE_BINARY_DIR: C:/opt/eudaq/build
-- PROJECT_SOURCE_DIR: C:/opt/eudaq
-- This is CMake 3.0.1 (C:/Program Files/CMake/bin/cmake.exe) running on Windows
 (6.1 )
-- 32-bit compiler detected
-- Using MSVC (C:/Program Files/Microsoft Visual Studio 12.0/VC/bin/cl.exe) vers
ion 18.0.21005.1
-- Compiler Flags:  /DWIN32 /D_WINDOWS /W3 /GR /EHsc;-fPIC -O2  /D "CPP11"
-- With COMPILE_DEFINITIONS = CPP11=1;EUDAQ_PLATFORM=PF_WIN32;EUDAQ_FUNC=__FUNCT
ION__
-- Could NOT find PythonInterp (missing:  PYTHON_EXECUTABLE)
-- *** Build Type  ***
-- *** Compiler Flags:   /DWIN32 /D_WINDOWS /W3 /GR /EHsc;-fPIC -O2  /D "CPP11"
-- *** Compiler Flags:  -fPIC -O2
-- Configuring done
-- Generating done
-- Build files have been written to: C:/opt/eudaq/build
```

BUILDING 

```
c:\opt\eudaq\build>MSBUILD.exe INSTALL.vcxproj /p:Configuration=Release

[...]

  Mupix3ConverterPlugin.cc
  Mupix4ConverterPlugin.cc
C:\opt\eudaq\.\main\include\eudaq/Mupix.hh(107): warning C4800: 'const uint8_t'
 : forcing value to bool 'true' or 'false' (performance warning) [C:\opt\eudaq\
build\main\lib\EUDAQ.vcxproj]
  MVDConverterPlugin.cc
  NIConverterPlugin.cc
  PyBARConverterPlugin.cc
  SCTConverterPlugin.cc
  Generating Code...
  Compiling...
  SCTUpgradeConverterPlugin.cc
  TAKIConverterPlugin.cc
  TestConverterPlugin.cc
  TimepixConverterPlugin.cc
  TLUConverterPlugin.cc
  USBpixConverterPlugin.cc
  USBPixI4ConverterPlugin.cc
  Generating Code...
Done Building Project "C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj" (default targ
ets) -- FAILED.

Done Building Project "C:\opt\eudaq\build\main\exe\ClusterExtractor.exe.vcxproj
" (default targets) -- FAILED.

Done Building Project "C:\opt\eudaq\build\ALL_BUILD.vcxproj" (default targets)
-- FAILED.

Done Building Project "c:\opt\eudaq\build\INSTALL.vcxproj" (default targets) --
 FAILED.


Build FAILED.

"c:\opt\eudaq\build\INSTALL.vcxproj" (default target) (1) ->
"C:\opt\eudaq\build\ALL_BUILD.vcxproj" (default target) (2) ->
"C:\opt\eudaq\build\main\exe\ClusterExtractor.exe.vcxproj" (default target) (3)
 ->
"C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj" (default target) (4) ->
(ClCompile target) ->
  cl : Command line warning D9002: ignoring unknown option '/EH;' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  cl : Command line warning D9002: ignoring unknown option '/EH-' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  cl : Command line warning D9002: ignoring unknown option '/EHf' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  cl : Command line warning D9002: ignoring unknown option '/EHP' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  cl : Command line warning D9002: ignoring unknown option '/EHI' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  cl : Command line warning D9002: ignoring unknown option '/EHC' [C:\opt\eudaq
\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/AidaFileReader.hh(20): warning C4244: 'retu
rn' : conversion from 'const unsigned __int64' to 'unsigned int', possible loss
 of data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\AidaPacket.cc(46): warning C4244: 'argument' : conversi
on from 'const uint64_t' to 'size_t', possible loss of data [C:\opt\eudaq\build
\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\AidaPacket.cc(52): warning C4244: 'initializing' : conv
ersion from 'uint64_t' to 'unsigned int', possible loss of data [C:\opt\eudaq\b
uild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\AidaPacket.cc(53): warning C4244: 'argument' : conversi
on from 'uint64_t' to 'size_t', possible loss of data [C:\opt\eudaq\build\main\
lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\AidaPacket.cc(156): warning C4244: 'initializing' : con
version from 'uint64_t' to 'int', possible loss of data [C:\opt\eudaq\build\mai
n\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\AidaPacket.cc(172): warning C4244: 'argument' : convers
ion from 'uint64_t' to 'const int', possible loss of data [C:\opt\eudaq\build\m
ain\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\MetaData.cc(42): warning C4244: 'return' : conversion f
rom 'uint64_t' to 'int', possible loss of data [C:\opt\eudaq\build\main\lib\EUD
AQ.vcxproj]
  ..\..\..\main\lib\src\MetaData.cc(50): warning C4800: 'uint64_t' : forcing va
lue to bool 'true' or 'false' (performance warning) [C:\opt\eudaq\build\main\li
b\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(160): warning C4800: 'int'
: forcing value to bool 'true' or 'false' (performance warning) [C:\opt\eudaq\b
uild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(235): warning C4244: '=' :
conversion from 'uint64_t' to 'unsigned int', possible loss of data [C:\opt\eud
aq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1093): warning C4244: '+='
: conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bu
ild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1094): warning C4244: '+='
: conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bu
ild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1095): warning C4244: '+='
: conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bu
ild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1096): warning C4244: '+='
: conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bu
ild\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1112): warning C4244: '=' :
 conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bui
ld\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1117): warning C4244: '=' :
 conversion from 'Double_t' to 'float', possible loss of data [C:\opt\eudaq\bui
ld\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1118): warning C4244: '/='
: conversion from 'double' to 'float', possible loss of data [C:\opt\eudaq\buil
d\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\PALPIDEFSConverterPlugin.cc(1119): warning C4244: '/='
: conversion from 'double' to 'float', possible loss of data [C:\opt\eudaq\buil
d\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/Serializer.hh(207): warning C4800: 'unsigne
d char' : forcing value to bool 'true' or 'false' (performance warning) [C:\opt
\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\src\Utils.cc(196): warning C4800: 'int' : forcing value to
bool 'true' or 'false' (performance warning) [C:\opt\eudaq\build\main\lib\EUDAQ
.vcxproj]
  ..\..\..\main\lib\plugins\EUDRBConverterPlugin.cc(341): warning C4800: 'int'
: forcing value to bool 'true' or 'false' (performance warning) [C:\opt\eudaq\b
uild\main\lib\EUDAQ.vcxproj]
  c:\root\include\TPad.h(323): warning C4800: 'Int_t' : forcing value to bool '
true' or 'false' (performance warning) [C:\opt\eudaq\build\main\lib\EUDAQ.vcxpr
oj]
  c:\root\include\TPad.h(324): warning C4800: 'Int_t' : forcing value to bool '
true' or 'false' (performance warning) [C:\opt\eudaq\build\main\lib\EUDAQ.vcxpr
oj]
  c:\root\include\TPad.h(325): warning C4800: 'Int_t' : forcing value to bool '
true' or 'false' (performance warning) [C:\opt\eudaq\build\main\lib\EUDAQ.vcxpr
oj]
  c:\root\include\TPad.h(381): warning C4800: 'TVirtualViewer3D *const ' : forc
ing value to bool 'true' or 'false' (performance warning) [C:\opt\eudaq\build\m
ain\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(47): warni
ng C4305: 'initializing' : truncation from 'double' to 'const float' [C:\opt\eu
daq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(63): warni
ng C4244: 'return' : conversion from 'double' to 'Float_t', possible loss of da
ta [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(80): warni
ng C4244: 'return' : conversion from 'double' to 'Float_t', possible loss of da
ta [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(87): warni
ng C4244: 'return' : conversion from 'double' to 'Float_t', possible loss of da
ta [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(96): warni
ng C4244: 'return' : conversion from 'double' to 'Float_t', possible loss of da
ta [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(107): warn
ing C4244: 'argument' : conversion from 'double' to 'float', possible loss of d
ata [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(189): warn
ing C4244: 'initializing' : conversion from 'double' to 'int', possible loss of
 data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(190): warn
ing C4244: 'initializing' : conversion from 'double' to 'int', possible loss of
 data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(223): warn
ing C4244: 'initializing' : conversion from 'Double_t' to 'Float_t', possible l
oss of data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(224): warn
ing C4244: 'initializing' : conversion from 'Double_t' to 'Float_t', possible l
oss of data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(225): warn
ing C4244: 'return' : conversion from 'double' to 'Float_t', possible loss of d
ata [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(257): warn
ing C4244: 'initializing' : conversion from 'double' to 'Float_t', possible los
s of data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(266): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(278): warn
ing C4244: 'return' : conversion from 'double' to 'float', possible loss of dat
a [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(354): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(355): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(356): warn
ing C4244: '=' : conversion from 'Int_t' to 'Float_t', possible loss of data [C
:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(357): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(360): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(361): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(362): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(363): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(364): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(365): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(366): warn
ing C4244: '=' : conversion from 'Int_t' to 'Float_t', possible loss of data [C
:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(367): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(373): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(374): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(375): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(376): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(377): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(378): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(379): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(380): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(381): warn
ing C4244: '=' : conversion from 'Int_t' to 'Float_t', possible loss of data [C
:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(382): warn
ing C4244: '=' : conversion from 'Double_t' to 'Float_t', possible loss of data
 [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(564): warn
ing C4305: 'initializing' : truncation from 'double' to 'float' [C:\opt\eudaq\b
uild\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(579): warn
ing C4244: '=' : conversion from 'double' to 'float', possible loss of data [C:
\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(614): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(615): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(616): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(617): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(618): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(619): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(666): warn
ing C4244: 'initializing' : conversion from 'double' to 'Float_t', possible los
s of data [C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/ExplorerConverterPluginHelpers.h(670): warn
ing C4244: '=' : conversion from 'Double_t' to 'float', possible loss of data [
C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\plugins\ExplorerConverterPlugin.cc(467): warning C4244: 'in
itializing' : conversion from 'double' to 'short', possible loss of data [C:\op
t\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\plugins\ExplorerConverterPlugin.cc(476): warning C4244: 'in
itializing' : conversion from 'double' to 'short', possible loss of data [C:\op
t\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\plugins\ExplorerConverterPlugin.cc(509): warning C4244: 'in
itializing' : conversion from 'double' to 'short', possible loss of data [C:\op
t\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\plugins\ExplorerConverterPlugin.cc(544): warning C4244: 'in
itializing' : conversion from 'double' to 'short', possible loss of data [C:\op
t\eudaq\build\main\lib\EUDAQ.vcxproj]
  ..\..\..\main\lib\plugins\ExplorerConverterPlugin.cc(548): warning C4244: 'in
itializing' : conversion from 'double' to 'short', possible loss of data [C:\op
t\eudaq\build\main\lib\EUDAQ.vcxproj]
  C:\opt\eudaq\.\main\include\eudaq/Mupix.hh(107): warning C4800: 'const uint8_
t' : forcing value to bool 'true' or 'false' (performance warning) [C:\opt\euda
q\build\main\lib\EUDAQ.vcxproj]


"c:\opt\eudaq\build\INSTALL.vcxproj" (default target) (1) ->
"C:\opt\eudaq\build\ALL_BUILD.vcxproj" (default target) (2) ->
"C:\opt\eudaq\build\main\exe\ClusterExtractor.exe.vcxproj" (default target) (3)
 ->
"C:\opt\eudaq\build\main\lib\EUDAQ.vcxproj" (default target) (4) ->
(ClCompile target) ->
  c:\root\include\TVirtualX.h(202): error C2059: syntax error : 'constant' [C:\
opt\eudaq\build\main\lib\EUDAQ.vcxproj]

    83 Warning(s)
    1 Error(s)

Time Elapsed 00:02:15.30
```

**Comment by dreyling** [comment]
(Sorry, I forgot to post on which OS. I ve updated the first entry.)

**Comment by arummler** [comment]
Did you pull a fresh version? This error should be solved with commit 8e375216bf0e4a008f6fa4f4267bc4d48e3871f5 which was merged yesterday into 1.5-dev. At least now it works for me on Windows 7. Only the OnlineMonitor is still crashing during the run time, I am currently trying to investigate, what I found up to now I attached to #17.

**Comment by dreyling** [comment]
Yes, now it is working: I've ckecked out v1.5-dev and EUDAQ compiles.
